CSharp/BatchExplorer/Models/ComputeNodeModel.cs

```diff
@@ -212,6 +219,47 @@ public override List<PropertyModel> PropertyModel
         #region Operations on ComputeNodes
 
         /// <summary>
+        /// Puts the compute node online
+        /// </summary>
+        public async Task DisableAsync()
```

Comment:
I would call this "DisableSchedulingAsync" and "EnableSchedulingAsync" personally.  "Disabling" a compute note could have multiple connotations I think...